src/rocks_record_store.cpp

```diff
@@ -1190,7 +1190,6 @@ namespace mongo {
     }
 
     boost::optional<Record> RocksRecordStore::Cursor::curr() {
-        invariantRocksOK(_iterator->status());
         if (!_iterator->Valid()) {
             _eof = true;
```

Comment:
`_iterator->status()`'s time complexity is `O(n)`, which `n` is the number of sorted runs, when we have many level0 SST files(we disabled compaction during intial sync), `_iterator->status()` produce a greate performance degrade.